src/main/java/duckie/Ui.java

```diff
@@ -0,0 +1,161 @@
+package duckie;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duckie.task.Task;
+
+/**
+ * Deals with the interaction with users
+ */
+public class Ui {
+    private static final String INDENT = "\t";
+    private static final String HORIZ_LINE = INDENT
+            + "____________________________________________________________";
+    private static final String LOGO = INDENT
+            + "           ____                   _      _\n"
+            + INDENT + "    __    |  _ \\   _   _    ___  | | _  (_)  ___     __\n"
+            + INDENT + "___( o)>  | | | | | | | | /  __| | |/ / | | / _ \\  <(o )___ \n"
+            + INDENT + "\\ <_. )   | |_| | | | | | | (__  |   <  | | | __/   ( ._> /\n"
+            + INDENT + " `---'    |____/  \\___,_|  \\ __| |_|\\_\\ |_| \\___|    `___' \n";
+    private Scanner sc;
+
+    /**
+     * Instantiate an Ui object with the Scanner object ready
+     */
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Returns the horizontal line required to break the lines
+     *
+     * @return the horizontal line String
+     */
+    public static String getHorizLine() {
+        return HORIZ_LINE;
+    }
+
+    /**
+     * Display the horizontal line to break the lines
+     */
+    public static void showLine() {
+        System.out.println(HORIZ_LINE);
+    }
```

Comment:
Sorry for pointing out a trivial issue but the previous function of getHorizLine would probably be enough for your code to function. Probably consider removing the method.